src/app/resources/browser/styles/tokens.css

```
@@ -0,0 +1,306 @@
+  --danger-foreground: var(--on-accent);
+  --success: var(--green);
+  --success-foreground: var(--on-accent);
+  --warning: var(--amber);
+  --warning-foreground: var(--on-accent);
+  --info: var(--blue);
+  --info-foreground: var(--on-accent);
+
+  --ring: var(--accent);
+
+  /* ── TYPE — families (Inter v4 body) ──────────────────────── */
+  --display: "Source Serif 4", "Lora", Georgia, serif; /* @kind font */
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)